static/app/components/feedback/feedbackItem/feedbackActions.tsx

```diff
@@ -35,7 +35,7 @@ export default function FeedbackActions({
   }
 
   return (
-    <Flex gap={space(1)} align="center" className={className} style={style}>
+    <Flex gap={space(1)} align="flex-end" className={className} style={style}>
```

Comment:
center seemed ok to me. idk

Reply:
https://github.com/user-attachments/assets/266dea68-c461-4307-a0b4-e6a4a5951be7

it's a super small difference😭